targets/targets.json

```diff
@@ -2460,7 +2460,7 @@
         "supported_toolchains": ["uARM", "ARM"],
         "inherits": ["Target"],
         "device_has": ["ANALOGIN", "I2C", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "RTC", "SERIAL", "SPI", "SPISLAVE", "STDIO_MESSAGES"],
-        "release_versions": ["2"]
+        "release_versions": ["2", "5"]
```

Comment:
You must support ARM, GCC_ARM, and the IAR compilers to enable the release version "5" (same goes for the below changes)